apps/src/JSInterpreter.js

```diff
@@ -434,8 +435,9 @@ JSInterpreter.prototype.executeInterpreter = function (firstStep, runUntilCallba
     var err = safeStepInterpreter(this);
     if (!err) {
       var nodeType = this.interpreter.stateStack[0].node.type;
+      this.atInterstitialNode = INTERSTITIAL_NODES.hasOwnProperty(nodeType);
```

Comment:
Hmm this won't matter for `Program` or `BlockStatement` (both can be fast-forwarded), but it looks like it does prevent us from breaking on the start of a `SwitchStatement`.

I can either check `state.n_ > 0`, or remove `SwitchStatement` from the list of interstitial nodes.

(I'll fix this in the parent PR: https://github.com/code-dot-org/code-dot-org/pull/6753)